ajax/libs/wangEditor/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "wangEditor",
+  "filename": "js/wangEditor.min.js",
+  "version": "2.0.15.1",
+  "description": "wangEditor - 基于javascript和css开发的 Web富文本编辑器， 轻量、简洁、易用、开源免费",
+  "homepage": "http://wangeditor.github.io/",
+  "keywords": ["wangEditor", "富文本编辑器"],
+  "maintainers": [
+    {
+      "name": "wangfupeng",
+      "web": "http://www.cnblogs.com/wangfupeng1988/default.html?OnlyTitle=1",
+      "mail": "wangfupeng1988@163.com"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/wangfupeng1988/wangEditor"
+    }
+  ]
+}
```

Comment:
Please add npm auto-update config.
You could take a look at [this](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).
Thanks for your help!